computas/tekniskmuseum-backend
# Implement allowedFile function

# Changes :gem:
* Implement the allowedFile function with the following constraints: :snail:
  * The image can't have a resolution smaller than 256x256
  * The image needs to be a png file
  * The image can't be larger than 4 MB
* Add test data to test the allowedFile function :turtle:
* Implement test functions to verify the allowedFile function:
  * Use too large image
  * Use image with too low resolution
  * Use JPG image

# Notes :book: :pencil2:
_More tests may be added in the future. The PNG file is only verified by checking the end of the filename. Verification of the format may also be necessary._
